Makefile

```
@@ -27,9 +27,10 @@ test:
 .PHONY: integration-test
 # Run integration tests
 integration-test: cli-fast
+	mkdir -p reports/
 	go get github.com/jstemmer/go-junit-report
-	go test -tags integration ./pkg/... ./cmd/... -v -mod=readonly -coverprofile cover-integration.out 2>&1 | go-junit-report -set-exit-code > reports/integration_report.xml
-	go run ./cmd/kubectl-kudo test 2>&1 | go-junit-report -set-exit-code > reports/kudo_test_report.xml
+	go test -tags integration ./pkg/... ./cmd/... -v -mod=readonly -coverprofile cover-integration.out 2>&1 |tee /dev/fd/2 |go-junit-report -set-exit-code > reports/integration_report.xml
```

```suggestion
go test -tags integration ./pkg/... ./cmd/... -v -mod=readonly -coverprofile cover-integration.out 2>&1 | tee /dev/fd/2 | go-junit-report -set-exit-code > reports/integration_report.xml
```

Nitpick: Space before and after a pipe operator to be consistent.